http/services.go

```diff
@@ -0,0 +1,53 @@
+package http
+
+import (
+	"context"
+	"raccoon/collection"
+	"raccoon/logger"
+
+	"raccoon/http/grpc"
+	"raccoon/http/pprof"
+	"raccoon/http/rest"
+)
+
+type bootstrapper interface {
+	// Init initialize each HTTP based server. Return error if initialization failed. Put the Serve() function as return mostly suffice for Init process.
+	Init() error
```

Comment:
Can we pass context as a function param here too? Just to make it standard.

Reply:
Done